setup.py

```diff
@@ -10,7 +10,8 @@
 remote_deps = ["rpyc==4.1.5", "cloudpickle", "boto3"]
 spreadsheet_deps = ["modin-spreadsheet>=0.1.0"]
 sql_deps = ["dfsql>=0.4.2", "pyparsing<=2.4.7"]
-all_deps = dask_deps + ray_deps + remote_deps + spreadsheet_deps
+fuzzydata_deps = ["fuzzydata>=0.0.3"]
```

Comment:
Probably not necessary to add for the `setup.py` because it is used for development.

Reply:
Makes sense!